## `nixpkgs-review` result

Generated using [`nixpkgs-review-gha`](https://github.com/Defelo/nixpkgs-review-gha)

Command: `nixpkgs-review pr 499867`
Commit: [`572b8eddfaf21565d2846e2874a42425202fbde3`](https://github.com/NixOS/nixpkgs/commit/572b8eddfaf21565d2846e2874a42425202fbde3) ([subsequent changes](https://github.com/NixOS/nixpkgs/compare/572b8eddfaf21565d2846e2874a42425202fbde3..pull/499867/head))
Merge: [`8f929cdbdc951d23dd40d46d39be1c8b1fda272a`](https://github.com/NixOS/nixpkgs/commit/8f929cdbdc951d23dd40d46d39be1c8b1fda272a)

Logs: https://github.com/yzhou216/nixpkgs-review-gha/actions/runs/23105470088

<details><summary>Download packages from cache:</summary><ul>
<li><details><summary><code>x86_64-linux</code></summary>

```shell
nix-store -r --add-root nixpkgs-pr-499867-x86_64-linux \
  --option binary-caches 'https://cache.nixos.org/ https://yzhou216.cachix.org' \
  --option trusted-public-keys '
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  yzhou216.cachix.org-1:O7nGwQnLGGBGE8zYFMipVzPVN956FcV57y9SqwEP+O8=
  ' \
  /nix/store/gv4hd9g13qpalrpfs9rr4vdbk3yc5y95-cataclysm-dda-0.H-2025-07-10-0402
```
</details></li>
<li><details><summary><code>aarch64-linux</code></summary>

```shell
nix-store -r --add-root nixpkgs-pr-499867-aarch64-linux \
  --option binary-caches 'https://cache.nixos.org/ https://yzhou216.cachix.org' \
  --option trusted-public-keys '
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  yzhou216.cachix.org-1:O7nGwQnLGGBGE8zYFMipVzPVN956FcV57y9SqwEP+O8=
  ' \
  /nix/store/lhbnd98dlsgj8pl4qlv2pbxza9p89h45-cataclysm-dda-0.H-2025-07-10-0402
```
</details></li>
</ul></details>


---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>cataclysm-dda</li>
  </ul>
</details>

---
### `aarch64-linux`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>cataclysm-dda</li>
  </ul>
</details>

---
### `x86_64-darwin` (sandbox = true)
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>cataclysm-dda</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-darwin`</summary>
<details>
<summary>cataclysm-dda</summary>
<pre>clang++ -Isrc -isystem src/third-party -DRELEASE -DMACOSX -DOSX_SDL2_LIBS -DTILES -DBACKTRACE -DLOCALIZE -DPREFIX=&quot;/nix/store/yjgqfq330h8sijs5jzfifym2i9av47h1-cataclysm-dda-0.H-2025-07-10-0402&quot; -DDATA_DIR_PREFIX -DUSE_HOME_DIR -Os -Wodr -Werror -Wall -Wextra -Wformat-signedness -Wlogical-op -Wmissing-declarations -Wmissing-noreturn -Wnon-virtual-dtor -Wold-style-cast -Woverloaded-virtual -Wpedantic -Wsuggest-override -Wunused-macros -Wzero-as-null-pointer-constant -Wno-unknown-warning-option -Wno-dangling-reference -Wno-c++20-compat -Wredundant-decls  -g1  -fsigned-char -stdlib=libc++  -std=c++17 -mmacosx-version-min=10.13 -D_THREAD_SAFE -I/nix/store/fxy45bn05m1rr3vhlyhr6b8gkv56kp77-sdl2-compat-2.32.64-dev/include -I/nix/store/fxy45bn05m1rr3vhlyhr6b8gkv56kp77-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/3wzwvx7bf5frbx2yg7spdkqdyn36kasw-SDL2_mixer-2.8.1-dev/include/SDL2 -DSDL_SOUND -I/SDL2 -I/nix/store/fxy45bn05m1rr3vhlyhr6b8gkv56kp77-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/d9xn4qi7xbkyh89h1bmsms179i4rlk4a-SDL2_image-2.8.8/include/SDL2 -I/nix/store/3wzwvx7bf5frbx2yg7spdkqdyn36kasw-SDL2_mixer-2.8.1-dev/include/SDL2 -I/nix/store/1c1zckkvp52jsalvv27arlhsk8g93sfr-SDL2_ttf-2.24.0/include/SDL2 -I/nix/store/fxy45bn05m1rr3vhlyhr6b8gkv56kp77-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/d9xn4qi7xbkyh89h1bmsms179i4rlk4a-SDL2_image-2.8.8/include/SDL2 -I/nix/store/3wzwvx7bf5frbx2yg7spdkqdyn36kasw-SDL2_mixer-2.8.1-dev/include/SDL2 -I/nix/store/1c1zckkvp52jsalvv27arlhsk8g93sfr-SDL2_ttf-2.24.0/include/SDL2 -D_THREAD_SAFE -I/ -MMD -MP -Winvalid-pch -include-pch pch/main-pch.hpp.pch -c src/map_selector.cpp -o obj/tiles/map_selector.o
clang++ -Isrc -isystem src/third-party -DRELEASE -DMACOSX -DOSX_SDL2_LIBS -DTILES -DBACKTRACE -DLOCALIZE -DPREFIX=&quot;/nix/store/yjgqfq330h8sijs5jzfifym2i9av47h1-cataclysm-dda-0.H-2025-07-10-0402&quot; -DDATA_DIR_PREFIX -DUSE_HOME_DIR -Os -Wodr -Werror -Wall -Wextra -Wformat-signedness -Wlogical-op -Wmissing-declarations -Wmissing-noreturn -Wnon-virtual-dtor -Wold-style-cast -Woverloaded-virtual -Wpedantic -Wsuggest-override -Wunused-macros -Wzero-as-null-pointer-constant -Wno-unknown-warning-option -Wno-dangling-reference -Wno-c++20-compat -Wredundant-decls  -g1  -fsigned-char -stdlib=libc++  -std=c++17 -mmacosx-version-min=10.13 -D_THREAD_SAFE -I/nix/store/fxy45bn05m1rr3vhlyhr6b8gkv56kp77-sdl2-compat-2.32.64-dev/include -I/nix/store/fxy45bn05m1rr3vhlyhr6b8gkv56kp77-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/3wzwvx7bf5frbx2yg7spdkqdyn36kasw-SDL2_mixer-2.8.1-dev/include/SDL2 -DSDL_SOUND -I/SDL2 -I/nix/store/fxy45bn05m1rr3vhlyhr6b8gkv56kp77-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/d9xn4qi7xbkyh89h1bmsms179i4rlk4a-SDL2_image-2.8.8/include/SDL2 -I/nix/store/3wzwvx7bf5frbx2yg7spdkqdyn36kasw-SDL2_mixer-2.8.1-dev/include/SDL2 -I/nix/store/1c1zckkvp52jsalvv27arlhsk8g93sfr-SDL2_ttf-2.24.0/include/SDL2 -I/nix/store/fxy45bn05m1rr3vhlyhr6b8gkv56kp77-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/d9xn4qi7xbkyh89h1bmsms179i4rlk4a-SDL2_image-2.8.8/include/SDL2 -I/nix/store/3wzwvx7bf5frbx2yg7spdkqdyn36kasw-SDL2_mixer-2.8.1-dev/include/SDL2 -I/nix/store/1c1zckkvp52jsalvv27arlhsk8g93sfr-SDL2_ttf-2.24.0/include/SDL2 -D_THREAD_SAFE -I/ -MMD -MP -Winvalid-pch -include-pch pch/main-pch.hpp.pch -c src/mapbuffer.cpp -o obj/tiles/mapbuffer.o
clang++ -Isrc -isystem src/third-party -DRELEASE -DMACOSX -DOSX_SDL2_LIBS -DTILES -DBACKTRACE -DLOCALIZE -DPREFIX=&quot;/nix/store/yjgqfq330h8sijs5jzfifym2i9av47h1-cataclysm-dda-0.H-2025-07-10-0402&quot; -DDATA_DIR_PREFIX -DUSE_HOME_DIR -Os -Wodr -Werror -Wall -Wextra -Wformat-signedness -Wlogical-op -Wmissing-declarations -Wmissing-noreturn -Wnon-virtual-dtor -Wold-style-cast -Woverloaded-virtual -Wpedantic -Wsuggest-override -Wunused-macros -Wzero-as-null-pointer-constant -Wno-unknown-warning-option -Wno-dangling-reference -Wno-c++20-compat -Wredundant-decls  -g1  -fsigned-char -stdlib=libc++  -std=c++17 -mmacosx-version-min=10.13 -D_THREAD_SAFE -I/nix/store/fxy45bn05m1rr3vhlyhr6b8gkv56kp77-sdl2-compat-2.32.64-dev/include -I/nix/store/fxy45bn05m1rr3vhlyhr6b8gkv56kp77-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/3wzwvx7bf5frbx2yg7spdkqdyn36kasw-SDL2_mixer-2.8.1-dev/include/SDL2 -DSDL_SOUND -I/SDL2 -I/nix/store/fxy45bn05m1rr3vhlyhr6b8gkv56kp77-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/d9xn4qi7xbkyh89h1bmsms179i4rlk4a-SDL2_image-2.8.8/include/SDL2 -I/nix/store/3wzwvx7bf5frbx2yg7spdkqdyn36kasw-SDL2_mixer-2.8.1-dev/include/SDL2 -I/nix/store/1c1zckkvp52jsalvv27arlhsk8g93sfr-SDL2_ttf-2.24.0/include/SDL2 -I/nix/store/fxy45bn05m1rr3vhlyhr6b8gkv56kp77-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/d9xn4qi7xbkyh89h1bmsms179i4rlk4a-SDL2_image-2.8.8/include/SDL2 -I/nix/store/3wzwvx7bf5frbx2yg7spdkqdyn36kasw-SDL2_mixer-2.8.1-dev/include/SDL2 -I/nix/store/1c1zckkvp52jsalvv27arlhsk8g93sfr-SDL2_ttf-2.24.0/include/SDL2 -D_THREAD_SAFE -I/ -MMD -MP -Winvalid-pch -include-pch pch/main-pch.hpp.pch -c src/mapdata.cpp -o obj/tiles/mapdata.o
clang++ -Isrc -isystem src/third-party -DRELEASE -DMACOSX -DOSX_SDL2_LIBS -DTILES -DBACKTRACE -DLOCALIZE -DPREFIX=&quot;/nix/store/yjgqfq330h8sijs5jzfifym2i9av47h1-cataclysm-dda-0.H-2025-07-10-0402&quot; -DDATA_DIR_PREFIX -DUSE_HOME_DIR -Os -Wodr -Werror -Wall -Wextra -Wformat-signedness -Wlogical-op -Wmissing-declarations -Wmissing-noreturn -Wnon-virtual-dtor -Wold-style-cast -Woverloaded-virtual -Wpedantic -Wsuggest-override -Wunused-macros -Wzero-as-null-pointer-constant -Wno-unknown-warning-option -Wno-dangling-reference -Wno-c++20-compat -Wredundant-decls  -g1  -fsigned-char -stdlib=libc++  -std=c++17 -mmacosx-version-min=10.13 -D_THREAD_SAFE -I/nix/store/fxy45bn05m1rr3vhlyhr6b8gkv56kp77-sdl2-compat-2.32.64-dev/include -I/nix/store/fxy45bn05m1rr3vhlyhr6b8gkv56kp77-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/3wzwvx7bf5frbx2yg7spdkqdyn36kasw-SDL2_mixer-2.8.1-dev/include/SDL2 -DSDL_SOUND -I/SDL2 -I/nix/store/fxy45bn05m1rr3vhlyhr6b8gkv56kp77-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/d9xn4qi7xbkyh89h1bmsms179i4rlk4a-SDL2_image-2.8.8/include/SDL2 -I/nix/store/3wzwvx7bf5frbx2yg7spdkqdyn36kasw-SDL2_mixer-2.8.1-dev/include/SDL2 -I/nix/store/1c1zckkvp52jsalvv27arlhsk8g93sfr-SDL2_ttf-2.24.0/include/SDL2 -I/nix/store/fxy45bn05m1rr3vhlyhr6b8gkv56kp77-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/d9xn4qi7xbkyh89h1bmsms179i4rlk4a-SDL2_image-2.8.8/include/SDL2 -I/nix/store/3wzwvx7bf5frbx2yg7spdkqdyn36kasw-SDL2_mixer-2.8.1-dev/include/SDL2 -I/nix/store/1c1zckkvp52jsalvv27arlhsk8g93sfr-SDL2_ttf-2.24.0/include/SDL2 -D_THREAD_SAFE -I/ -MMD -MP -Winvalid-pch -include-pch pch/main-pch.hpp.pch -c src/mapgen.cpp -o obj/tiles/mapgen.o
clang++ -Isrc -isystem src/third-party -DRELEASE -DMACOSX -DOSX_SDL2_LIBS -DTILES -DBACKTRACE -DLOCALIZE -DPREFIX=&quot;/nix/store/yjgqfq330h8sijs5jzfifym2i9av47h1-cataclysm-dda-0.H-2025-07-10-0402&quot; -DDATA_DIR_PREFIX -DUSE_HOME_DIR -Os -Wodr -Werror -Wall -Wextra -Wformat-signedness -Wlogical-op -Wmissing-declarations -Wmissing-noreturn -Wnon-virtual-dtor -Wold-style-cast -Woverloaded-virtual -Wpedantic -Wsuggest-override -Wunused-macros -Wzero-as-null-pointer-constant -Wno-unknown-warning-option -Wno-dangling-reference -Wno-c++20-compat -Wredundant-decls  -g1  -fsigned-char -stdlib=libc++  -std=c++17 -mmacosx-version-min=10.13 -D_THREAD_SAFE -I/nix/store/fxy45bn05m1rr3vhlyhr6b8gkv56kp77-sdl2-compat-2.32.64-dev/include -I/nix/store/fxy45bn05m1rr3vhlyhr6b8gkv56kp77-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/3wzwvx7bf5frbx2yg7spdkqdyn36kasw-SDL2_mixer-2.8.1-dev/include/SDL2 -DSDL_SOUND -I/SDL2 -I/nix/store/fxy45bn05m1rr3vhlyhr6b8gkv56kp77-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/d9xn4qi7xbkyh89h1bmsms179i4rlk4a-SDL2_image-2.8.8/include/SDL2 -I/nix/store/3wzwvx7bf5frbx2yg7spdkqdyn36kasw-SDL2_mixer-2.8.1-dev/include/SDL2 -I/nix/store/1c1zckkvp52jsalvv27arlhsk8g93sfr-SDL2_ttf-2.24.0/include/SDL2 -I/nix/store/fxy45bn05m1rr3vhlyhr6b8gkv56kp77-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/d9xn4qi7xbkyh89h1bmsms179i4rlk4a-SDL2_image-2.8.8/include/SDL2 -I/nix/store/3wzwvx7bf5frbx2yg7spdkqdyn36kasw-SDL2_mixer-2.8.1-dev/include/SDL2 -I/nix/store/1c1zckkvp52jsalvv27arlhsk8g93sfr-SDL2_ttf-2.24.0/include/SDL2 -D_THREAD_SAFE -I/ -MMD -MP -Winvalid-pch -include-pch pch/main-pch.hpp.pch -c src/mapgen_functions.cpp -o obj/tiles/mapgen_functions.o
clang++ -Isrc -isystem src/third-party -DRELEASE -DMACOSX -DOSX_SDL2_LIBS -DTILES -DBACKTRACE -DLOCALIZE -DPREFIX=&quot;/nix/store/yjgqfq330h8sijs5jzfifym2i9av47h1-cataclysm-dda-0.H-2025-07-10-0402&quot; -DDATA_DIR_PREFIX -DUSE_HOME_DIR -Os -Wodr -Werror -Wall -Wextra -Wformat-signedness -Wlogical-op -Wmissing-declarations -Wmissing-noreturn -Wnon-virtual-dtor -Wold-style-cast -Woverloaded-virtual -Wpedantic -Wsuggest-override -Wunused-macros -Wzero-as-null-pointer-constant -Wno-unknown-warning-option -Wno-dangling-reference -Wno-c++20-compat -Wredundant-decls  -g1  -fsigned-char -stdlib=libc++  -std=c++17 -mmacosx-version-min=10.13 -D_THREAD_SAFE -I/nix/store/fxy45bn05m1rr3vhlyhr6b8gkv56kp77-sdl2-compat-2.32.64-dev/include -I/nix/store/fxy45bn05m1rr3vhlyhr6b8gkv56kp77-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/3wzwvx7bf5frbx2yg7spdkqdyn36kasw-SDL2_mixer-2.8.1-dev/include/SDL2 -DSDL_SOUND -I/SDL2 -I/nix/store/fxy45bn05m1rr3vhlyhr6b8gkv56kp77-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/d9xn4qi7xbkyh89h1bmsms179i4rlk4a-SDL2_image-2.8.8/include/SDL2 -I/nix/store/3wzwvx7bf5frbx2yg7spdkqdyn36kasw-SDL2_mixer-2.8.1-dev/include/SDL2 -I/nix/store/1c1zckkvp52jsalvv27arlhsk8g93sfr-SDL2_ttf-2.24.0/include/SDL2 -I/nix/store/fxy45bn05m1rr3vhlyhr6b8gkv56kp77-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/d9xn4qi7xbkyh89h1bmsms179i4rlk4a-SDL2_image-2.8.8/include/SDL2 -I/nix/store/3wzwvx7bf5frbx2yg7spdkqdyn36kasw-SDL2_mixer-2.8.1-dev/include/SDL2 -I/nix/store/1c1zckkvp52jsalvv27arlhsk8g93sfr-SDL2_ttf-2.24.0/include/SDL2 -D_THREAD_SAFE -I/ -MMD -MP -Winvalid-pch -include-pch pch/main-pch.hpp.pch -c src/mapgendata.cpp -o obj/tiles/mapgendata.o
clang++ -Isrc -isystem src/third-party -DRELEASE -DMACOSX -DOSX_SDL2_LIBS -DTILES -DBACKTRACE -DLOCALIZE -DPREFIX=&quot;/nix/store/yjgqfq330h8sijs5jzfifym2i9av47h1-cataclysm-dda-0.H-2025-07-10-0402&quot; -DDATA_DIR_PREFIX -DUSE_HOME_DIR -Os -Wodr -Werror -Wall -Wextra -Wformat-signedness -Wlogical-op -Wmissing-declarations -Wmissing-noreturn -Wnon-virtual-dtor -Wold-style-cast -Woverloaded-virtual -Wpedantic -Wsuggest-override -Wunused-macros -Wzero-as-null-pointer-constant -Wno-unknown-warning-option -Wno-dangling-reference -Wno-c++20-compat -Wredundant-decls  -g1  -fsigned-char -stdlib=libc++  -std=c++17 -mmacosx-version-min=10.13 -D_THREAD_SAFE -I/nix/store/fxy45bn05m1rr3vhlyhr6b8gkv56kp77-sdl2-compat-2.32.64-dev/include -I/nix/store/fxy45bn05m1rr3vhlyhr6b8gkv56kp77-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/3wzwvx7bf5frbx2yg7spdkqdyn36kasw-SDL2_mixer-2.8.1-dev/include/SDL2 -DSDL_SOUND -I/SDL2 -I/nix/store/fxy45bn05m1rr3vhlyhr6b8gkv56kp77-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/d9xn4qi7xbkyh89h1bmsms179i4rlk4a-SDL2_image-2.8.8/include/SDL2 -I/nix/store/3wzwvx7bf5frbx2yg7spdkqdyn36kasw-SDL2_mixer-2.8.1-dev/include/SDL2 -I/nix/store/1c1zckkvp52jsalvv27arlhsk8g93sfr-SDL2_ttf-2.24.0/include/SDL2 -I/nix/store/fxy45bn05m1rr3vhlyhr6b8gkv56kp77-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/d9xn4qi7xbkyh89h1bmsms179i4rlk4a-SDL2_image-2.8.8/include/SDL2 -I/nix/store/3wzwvx7bf5frbx2yg7spdkqdyn36kasw-SDL2_mixer-2.8.1-dev/include/SDL2 -I/nix/store/1c1zckkvp52jsalvv27arlhsk8g93sfr-SDL2_ttf-2.24.0/include/SDL2 -D_THREAD_SAFE -I/ -MMD -MP -Winvalid-pch -include-pch pch/main-pch.hpp.pch -c src/mapgenformat.cpp -o obj/tiles/mapgenformat.o
clang++ -Isrc -isystem src/third-party -DRELEASE -DMACOSX -DOSX_SDL2_LIBS -DTILES -DBACKTRACE -DLOCALIZE -DPREFIX=&quot;/nix/store/yjgqfq330h8sijs5jzfifym2i9av47h1-cataclysm-dda-0.H-2025-07-10-0402&quot; -DDATA_DIR_PREFIX -DUSE_HOME_DIR -Os -Wodr -Werror -Wall -Wextra -Wformat-signedness -Wlogical-op -Wmissing-declarations -Wmissing-noreturn -Wnon-virtual-dtor -Wold-style-cast -Woverloaded-virtual -Wpedantic -Wsuggest-override -Wunused-macros -Wzero-as-null-pointer-constant -Wno-unknown-warning-option -Wno-dangling-reference -Wno-c++20-compat -Wredundant-decls  -g1  -fsigned-char -stdlib=libc++  -std=c++17 -mmacosx-version-min=10.13 -D_THREAD_SAFE -I/nix/store/fxy45bn05m1rr3vhlyhr6b8gkv56kp77-sdl2-compat-2.32.64-dev/include -I/nix/store/fxy45bn05m1rr3vhlyhr6b8gkv56kp77-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/3wzwvx7bf5frbx2yg7spdkqdyn36kasw-SDL2_mixer-2.8.1-dev/include/SDL2 -DSDL_SOUND -I/SDL2 -I/nix/store/fxy45bn05m1rr3vhlyhr6b8gkv56kp77-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/d9xn4qi7xbkyh89h1bmsms179i4rlk4a-SDL2_image-2.8.8/include/SDL2 -I/nix/store/3wzwvx7bf5frbx2yg7spdkqdyn36kasw-SDL2_mixer-2.8.1-dev/include/SDL2 -I/nix/store/1c1zckkvp52jsalvv27arlhsk8g93sfr-SDL2_ttf-2.24.0/include/SDL2 -I/nix/store/fxy45bn05m1rr3vhlyhr6b8gkv56kp77-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/d9xn4qi7xbkyh89h1bmsms179i4rlk4a-SDL2_image-2.8.8/include/SDL2 -I/nix/store/3wzwvx7bf5frbx2yg7spdkqdyn36kasw-SDL2_mixer-2.8.1-dev/include/SDL2 -I/nix/store/1c1zckkvp52jsalvv27arlhsk8g93sfr-SDL2_ttf-2.24.0/include/SDL2 -D_THREAD_SAFE -I/ -MMD -MP -Winvalid-pch -include-pch pch/main-pch.hpp.pch -c src/mapsharing.cpp -o obj/tiles/mapsharing.o
clang++ -Isrc -isystem src/third-party -DRELEASE -DMACOSX -DOSX_SDL2_LIBS -DTILES -DBACKTRACE -DLOCALIZE -DPREFIX=&quot;/nix/store/yjgqfq330h8sijs5jzfifym2i9av47h1-cataclysm-dda-0.H-2025-07-10-0402&quot; -DDATA_DIR_PREFIX -DUSE_HOME_DIR -Os -Wodr -Werror -Wall -Wextra -Wformat-signedness -Wlogical-op -Wmissing-declarations -Wmissing-noreturn -Wnon-virtual-dtor -Wold-style-cast -Woverloaded-virtual -Wpedantic -Wsuggest-override -Wunused-macros -Wzero-as-null-pointer-constant -Wno-unknown-warning-option -Wno-dangling-reference -Wno-c++20-compat -Wredundant-decls  -g1  -fsigned-char -stdlib=libc++  -std=c++17 -mmacosx-version-min=10.13 -D_THREAD_SAFE -I/nix/store/fxy45bn05m1rr3vhlyhr6b8gkv56kp77-sdl2-compat-2.32.64-dev/include -I/nix/store/fxy45bn05m1rr3vhlyhr6b8gkv56kp77-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/3wzwvx7bf5frbx2yg7spdkqdyn36kasw-SDL2_mixer-2.8.1-dev/include/SDL2 -DSDL_SOUND -I/SDL2 -I/nix/store/fxy45bn05m1rr3vhlyhr6b8gkv56kp77-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/d9xn4qi7xbkyh89h1bmsms179i4rlk4a-SDL2_image-2.8.8/include/SDL2 -I/nix/store/3wzwvx7bf5frbx2yg7spdkqdyn36kasw-SDL2_mixer-2.8.1-dev/include/SDL2 -I/nix/store/1c1zckkvp52jsalvv27arlhsk8g93sfr-SDL2_ttf-2.24.0/include/SDL2 -I/nix/store/fxy45bn05m1rr3vhlyhr6b8gkv56kp77-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/d9xn4qi7xbkyh89h1bmsms179i4rlk4a-SDL2_image-2.8.8/include/SDL2 -I/nix/store/3wzwvx7bf5frbx2yg7spdkqdyn36kasw-SDL2_mixer-2.8.1-dev/include/SDL2 -I/nix/store/1c1zckkvp52jsalvv27arlhsk8g93sfr-SDL2_ttf-2.24.0/include/SDL2 -D_THREAD_SAFE -I/ -MMD -MP -Winvalid-pch -include-pch pch/main-pch.hpp.pch -c src/martialarts.cpp -o obj/tiles/martialarts.o
clang++ -Isrc -isystem src/third-party -DRELEASE -DMACOSX -DOSX_SDL2_LIBS -DTILES -DBACKTRACE -DLOCALIZE -DPREFIX=&quot;/nix/store/yjgqfq330h8sijs5jzfifym2i9av47h1-cataclysm-dda-0.H-2025-07-10-0402&quot; -DDATA_DIR_PREFIX -DUSE_HOME_DIR -Os -Wodr -Werror -Wall -Wextra -Wformat-signedness -Wlogical-op -Wmissing-declarations -Wmissing-noreturn -Wnon-virtual-dtor -Wold-style-cast -Woverloaded-virtual -Wpedantic -Wsuggest-override -Wunused-macros -Wzero-as-null-pointer-constant -Wno-unknown-warning-option -Wno-dangling-reference -Wno-c++20-compat -Wredundant-decls  -g1  -fsigned-char -stdlib=libc++  -std=c++17 -mmacosx-version-min=10.13 -D_THREAD_SAFE -I/nix/store/fxy45bn05m1rr3vhlyhr6b8gkv56kp77-sdl2-compat-2.32.64-dev/include -I/nix/store/fxy45bn05m1rr3vhlyhr6b8gkv56kp77-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/3wzwvx7bf5frbx2yg7spdkqdyn36kasw-SDL2_mixer-2.8.1-dev/include/SDL2 -DSDL_SOUND -I/SDL2 -I/nix/store/fxy45bn05m1rr3vhlyhr6b8gkv56kp77-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/d9xn4qi7xbkyh89h1bmsms179i4rlk4a-SDL2_image-2.8.8/include/SDL2 -I/nix/store/3wzwvx7bf5frbx2yg7spdkqdyn36kasw-SDL2_mixer-2.8.1-dev/include/SDL2 -I/nix/store/1c1zckkvp52jsalvv27arlhsk8g93sfr-SDL2_ttf-2.24.0/include/SDL2 -I/nix/store/fxy45bn05m1rr3vhlyhr6b8gkv56kp77-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/d9xn4qi7xbkyh89h1bmsms179i4rlk4a-SDL2_image-2.8.8/include/SDL2 -I/nix/store/3wzwvx7bf5frbx2yg7spdkqdyn36kasw-SDL2_mixer-2.8.1-dev/include/SDL2 -I/nix/store/1c1zckkvp52jsalvv27arlhsk8g93sfr-SDL2_ttf-2.24.0/include/SDL2 -D_THREAD_SAFE -I/ -MMD -MP -Winvalid-pch -include-pch pch/main-pch.hpp.pch -c src/material.cpp -o obj/tiles/material.o
clang++ -Isrc -isystem src/third-party -DRELEASE -DMACOSX -DOSX_SDL2_LIBS -DTILES -DBACKTRACE -DLOCALIZE -DPREFIX=&quot;/nix/store/yjgqfq330h8sijs5jzfifym2i9av47h1-cataclysm-dda-0.H-2025-07-10-0402&quot; -DDATA_DIR_PREFIX -DUSE_HOME_DIR -Os -Wodr -Werror -Wall -Wextra -Wformat-signedness -Wlogical-op -Wmissing-declarations -Wmissing-noreturn -Wnon-virtual-dtor -Wold-style-cast -Woverloaded-virtual -Wpedantic -Wsuggest-override -Wunused-macros -Wzero-as-null-pointer-constant -Wno-unknown-warning-option -Wno-dangling-reference -Wno-c++20-compat -Wredundant-decls  -g1  -fsigned-char -stdlib=libc++  -std=c++17 -mmacosx-version-min=10.13 -D_THREAD_SAFE -I/nix/store/fxy45bn05m1rr3vhlyhr6b8gkv56kp77-sdl2-compat-2.32.64-dev/include -I/nix/store/fxy45bn05m1rr3vhlyhr6b8gkv56kp77-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/3wzwvx7bf5frbx2yg7spdkqdyn36kasw-SDL2_mixer-2.8.1-dev/include/SDL2 -DSDL_SOUND -I/SDL2 -I/nix/store/fxy45bn05m1rr3vhlyhr6b8gkv56kp77-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/d9xn4qi7xbkyh89h1bmsms179i4rlk4a-SDL2_image-2.8.8/include/SDL2 -I/nix/store/3wzwvx7bf5frbx2yg7spdkqdyn36kasw-SDL2_mixer-2.8.1-dev/include/SDL2 -I/nix/store/1c1zckkvp52jsalvv27arlhsk8g93sfr-SDL2_ttf-2.24.0/include/SDL2 -I/nix/store/fxy45bn05m1rr3vhlyhr6b8gkv56kp77-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/d9xn4qi7xbkyh89h1bmsms179i4rlk4a-SDL2_image-2.8.8/include/SDL2 -I/nix/store/3wzwvx7bf5frbx2yg7spdkqdyn36kasw-SDL2_mixer-2.8.1-dev/include/SDL2 -I/nix/store/1c1zckkvp52jsalvv27arlhsk8g93sfr-SDL2_ttf-2.24.0/include/SDL2 -D_THREAD_SAFE -I/ -MMD -MP -Winvalid-pch -include-pch pch/main-pch.hpp.pch -c src/math_parser.cpp -o obj/tiles/math_parser.o
src/math_parser.cpp:632:13: error: function &#x27;operator()&#x27; could be declared with attribute &#x27;noreturn&#x27; [-Werror,-Wmissing-noreturn]
  632 |             []( auto /* v */ )
      |             ^
src/math_parser.cpp:765:9: error: function &#x27;operator()&#x27; could be declared with attribute &#x27;noreturn&#x27; [-Werror,-Wmissing-noreturn]
  765 |         []( auto /* v */ )
      |         ^
2 errors generated.
make: *** [Makefile:1057: obj/tiles/math_parser.o] Error 1
make: *** Waiting for unfinished jobs....</pre>
</details>
</details>

---
### `aarch64-darwin` (sandbox = true)
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>cataclysm-dda</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-darwin`</summary>
<details>
<summary>cataclysm-dda</summary>
<pre>clang++ -Isrc -isystem src/third-party -DRELEASE -DMACOSX -DOSX_SDL2_LIBS -DTILES -DBACKTRACE -DLOCALIZE -DPREFIX=&quot;/nix/store/f070rqbqlqrfklklqc0m93bbxsmqha2j-cataclysm-dda-0.H-2025-07-10-0402&quot; -DDATA_DIR_PREFIX -DUSE_HOME_DIR -Os -Wodr -Werror -Wall -Wextra -Wformat-signedness -Wlogical-op -Wmissing-declarations -Wmissing-noreturn -Wnon-virtual-dtor -Wold-style-cast -Woverloaded-virtual -Wpedantic -Wsuggest-override -Wunused-macros -Wzero-as-null-pointer-constant -Wno-unknown-warning-option -Wno-dangling-reference -Wno-c++20-compat -Wredundant-decls  -g1  -fsigned-char -stdlib=libc++  -std=c++17 -mmacosx-version-min=10.13 -D_THREAD_SAFE -I/nix/store/a1gcik4f5777g7yw2k2znjjmsadlgmcb-sdl2-compat-2.32.64-dev/include -I/nix/store/a1gcik4f5777g7yw2k2znjjmsadlgmcb-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/c16accmdcp1ackda871l4x0b234nhyxv-SDL2_mixer-2.8.1-dev/include/SDL2 -DSDL_SOUND -I/SDL2 -I/nix/store/a1gcik4f5777g7yw2k2znjjmsadlgmcb-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/2yc9y51272dzfd2vyfimwpg328db1p8n-SDL2_image-2.8.8/include/SDL2 -I/nix/store/c16accmdcp1ackda871l4x0b234nhyxv-SDL2_mixer-2.8.1-dev/include/SDL2 -I/nix/store/wki4knqr54n3zqgvl085i5ajhija37gg-SDL2_ttf-2.24.0/include/SDL2 -I/nix/store/a1gcik4f5777g7yw2k2znjjmsadlgmcb-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/2yc9y51272dzfd2vyfimwpg328db1p8n-SDL2_image-2.8.8/include/SDL2 -I/nix/store/c16accmdcp1ackda871l4x0b234nhyxv-SDL2_mixer-2.8.1-dev/include/SDL2 -I/nix/store/wki4knqr54n3zqgvl085i5ajhija37gg-SDL2_ttf-2.24.0/include/SDL2 -D_THREAD_SAFE -I/ -MMD -MP -Winvalid-pch -include-pch pch/main-pch.hpp.pch -c src/map_selector.cpp -o obj/tiles/map_selector.o
clang++ -Isrc -isystem src/third-party -DRELEASE -DMACOSX -DOSX_SDL2_LIBS -DTILES -DBACKTRACE -DLOCALIZE -DPREFIX=&quot;/nix/store/f070rqbqlqrfklklqc0m93bbxsmqha2j-cataclysm-dda-0.H-2025-07-10-0402&quot; -DDATA_DIR_PREFIX -DUSE_HOME_DIR -Os -Wodr -Werror -Wall -Wextra -Wformat-signedness -Wlogical-op -Wmissing-declarations -Wmissing-noreturn -Wnon-virtual-dtor -Wold-style-cast -Woverloaded-virtual -Wpedantic -Wsuggest-override -Wunused-macros -Wzero-as-null-pointer-constant -Wno-unknown-warning-option -Wno-dangling-reference -Wno-c++20-compat -Wredundant-decls  -g1  -fsigned-char -stdlib=libc++  -std=c++17 -mmacosx-version-min=10.13 -D_THREAD_SAFE -I/nix/store/a1gcik4f5777g7yw2k2znjjmsadlgmcb-sdl2-compat-2.32.64-dev/include -I/nix/store/a1gcik4f5777g7yw2k2znjjmsadlgmcb-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/c16accmdcp1ackda871l4x0b234nhyxv-SDL2_mixer-2.8.1-dev/include/SDL2 -DSDL_SOUND -I/SDL2 -I/nix/store/a1gcik4f5777g7yw2k2znjjmsadlgmcb-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/2yc9y51272dzfd2vyfimwpg328db1p8n-SDL2_image-2.8.8/include/SDL2 -I/nix/store/c16accmdcp1ackda871l4x0b234nhyxv-SDL2_mixer-2.8.1-dev/include/SDL2 -I/nix/store/wki4knqr54n3zqgvl085i5ajhija37gg-SDL2_ttf-2.24.0/include/SDL2 -I/nix/store/a1gcik4f5777g7yw2k2znjjmsadlgmcb-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/2yc9y51272dzfd2vyfimwpg328db1p8n-SDL2_image-2.8.8/include/SDL2 -I/nix/store/c16accmdcp1ackda871l4x0b234nhyxv-SDL2_mixer-2.8.1-dev/include/SDL2 -I/nix/store/wki4knqr54n3zqgvl085i5ajhija37gg-SDL2_ttf-2.24.0/include/SDL2 -D_THREAD_SAFE -I/ -MMD -MP -Winvalid-pch -include-pch pch/main-pch.hpp.pch -c src/mapbuffer.cpp -o obj/tiles/mapbuffer.o
clang++ -Isrc -isystem src/third-party -DRELEASE -DMACOSX -DOSX_SDL2_LIBS -DTILES -DBACKTRACE -DLOCALIZE -DPREFIX=&quot;/nix/store/f070rqbqlqrfklklqc0m93bbxsmqha2j-cataclysm-dda-0.H-2025-07-10-0402&quot; -DDATA_DIR_PREFIX -DUSE_HOME_DIR -Os -Wodr -Werror -Wall -Wextra -Wformat-signedness -Wlogical-op -Wmissing-declarations -Wmissing-noreturn -Wnon-virtual-dtor -Wold-style-cast -Woverloaded-virtual -Wpedantic -Wsuggest-override -Wunused-macros -Wzero-as-null-pointer-constant -Wno-unknown-warning-option -Wno-dangling-reference -Wno-c++20-compat -Wredundant-decls  -g1  -fsigned-char -stdlib=libc++  -std=c++17 -mmacosx-version-min=10.13 -D_THREAD_SAFE -I/nix/store/a1gcik4f5777g7yw2k2znjjmsadlgmcb-sdl2-compat-2.32.64-dev/include -I/nix/store/a1gcik4f5777g7yw2k2znjjmsadlgmcb-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/c16accmdcp1ackda871l4x0b234nhyxv-SDL2_mixer-2.8.1-dev/include/SDL2 -DSDL_SOUND -I/SDL2 -I/nix/store/a1gcik4f5777g7yw2k2znjjmsadlgmcb-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/2yc9y51272dzfd2vyfimwpg328db1p8n-SDL2_image-2.8.8/include/SDL2 -I/nix/store/c16accmdcp1ackda871l4x0b234nhyxv-SDL2_mixer-2.8.1-dev/include/SDL2 -I/nix/store/wki4knqr54n3zqgvl085i5ajhija37gg-SDL2_ttf-2.24.0/include/SDL2 -I/nix/store/a1gcik4f5777g7yw2k2znjjmsadlgmcb-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/2yc9y51272dzfd2vyfimwpg328db1p8n-SDL2_image-2.8.8/include/SDL2 -I/nix/store/c16accmdcp1ackda871l4x0b234nhyxv-SDL2_mixer-2.8.1-dev/include/SDL2 -I/nix/store/wki4knqr54n3zqgvl085i5ajhija37gg-SDL2_ttf-2.24.0/include/SDL2 -D_THREAD_SAFE -I/ -MMD -MP -Winvalid-pch -include-pch pch/main-pch.hpp.pch -c src/mapdata.cpp -o obj/tiles/mapdata.o
clang++ -Isrc -isystem src/third-party -DRELEASE -DMACOSX -DOSX_SDL2_LIBS -DTILES -DBACKTRACE -DLOCALIZE -DPREFIX=&quot;/nix/store/f070rqbqlqrfklklqc0m93bbxsmqha2j-cataclysm-dda-0.H-2025-07-10-0402&quot; -DDATA_DIR_PREFIX -DUSE_HOME_DIR -Os -Wodr -Werror -Wall -Wextra -Wformat-signedness -Wlogical-op -Wmissing-declarations -Wmissing-noreturn -Wnon-virtual-dtor -Wold-style-cast -Woverloaded-virtual -Wpedantic -Wsuggest-override -Wunused-macros -Wzero-as-null-pointer-constant -Wno-unknown-warning-option -Wno-dangling-reference -Wno-c++20-compat -Wredundant-decls  -g1  -fsigned-char -stdlib=libc++  -std=c++17 -mmacosx-version-min=10.13 -D_THREAD_SAFE -I/nix/store/a1gcik4f5777g7yw2k2znjjmsadlgmcb-sdl2-compat-2.32.64-dev/include -I/nix/store/a1gcik4f5777g7yw2k2znjjmsadlgmcb-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/c16accmdcp1ackda871l4x0b234nhyxv-SDL2_mixer-2.8.1-dev/include/SDL2 -DSDL_SOUND -I/SDL2 -I/nix/store/a1gcik4f5777g7yw2k2znjjmsadlgmcb-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/2yc9y51272dzfd2vyfimwpg328db1p8n-SDL2_image-2.8.8/include/SDL2 -I/nix/store/c16accmdcp1ackda871l4x0b234nhyxv-SDL2_mixer-2.8.1-dev/include/SDL2 -I/nix/store/wki4knqr54n3zqgvl085i5ajhija37gg-SDL2_ttf-2.24.0/include/SDL2 -I/nix/store/a1gcik4f5777g7yw2k2znjjmsadlgmcb-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/2yc9y51272dzfd2vyfimwpg328db1p8n-SDL2_image-2.8.8/include/SDL2 -I/nix/store/c16accmdcp1ackda871l4x0b234nhyxv-SDL2_mixer-2.8.1-dev/include/SDL2 -I/nix/store/wki4knqr54n3zqgvl085i5ajhija37gg-SDL2_ttf-2.24.0/include/SDL2 -D_THREAD_SAFE -I/ -MMD -MP -Winvalid-pch -include-pch pch/main-pch.hpp.pch -c src/mapgen.cpp -o obj/tiles/mapgen.o
clang++ -Isrc -isystem src/third-party -DRELEASE -DMACOSX -DOSX_SDL2_LIBS -DTILES -DBACKTRACE -DLOCALIZE -DPREFIX=&quot;/nix/store/f070rqbqlqrfklklqc0m93bbxsmqha2j-cataclysm-dda-0.H-2025-07-10-0402&quot; -DDATA_DIR_PREFIX -DUSE_HOME_DIR -Os -Wodr -Werror -Wall -Wextra -Wformat-signedness -Wlogical-op -Wmissing-declarations -Wmissing-noreturn -Wnon-virtual-dtor -Wold-style-cast -Woverloaded-virtual -Wpedantic -Wsuggest-override -Wunused-macros -Wzero-as-null-pointer-constant -Wno-unknown-warning-option -Wno-dangling-reference -Wno-c++20-compat -Wredundant-decls  -g1  -fsigned-char -stdlib=libc++  -std=c++17 -mmacosx-version-min=10.13 -D_THREAD_SAFE -I/nix/store/a1gcik4f5777g7yw2k2znjjmsadlgmcb-sdl2-compat-2.32.64-dev/include -I/nix/store/a1gcik4f5777g7yw2k2znjjmsadlgmcb-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/c16accmdcp1ackda871l4x0b234nhyxv-SDL2_mixer-2.8.1-dev/include/SDL2 -DSDL_SOUND -I/SDL2 -I/nix/store/a1gcik4f5777g7yw2k2znjjmsadlgmcb-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/2yc9y51272dzfd2vyfimwpg328db1p8n-SDL2_image-2.8.8/include/SDL2 -I/nix/store/c16accmdcp1ackda871l4x0b234nhyxv-SDL2_mixer-2.8.1-dev/include/SDL2 -I/nix/store/wki4knqr54n3zqgvl085i5ajhija37gg-SDL2_ttf-2.24.0/include/SDL2 -I/nix/store/a1gcik4f5777g7yw2k2znjjmsadlgmcb-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/2yc9y51272dzfd2vyfimwpg328db1p8n-SDL2_image-2.8.8/include/SDL2 -I/nix/store/c16accmdcp1ackda871l4x0b234nhyxv-SDL2_mixer-2.8.1-dev/include/SDL2 -I/nix/store/wki4knqr54n3zqgvl085i5ajhija37gg-SDL2_ttf-2.24.0/include/SDL2 -D_THREAD_SAFE -I/ -MMD -MP -Winvalid-pch -include-pch pch/main-pch.hpp.pch -c src/mapgen_functions.cpp -o obj/tiles/mapgen_functions.o
clang++ -Isrc -isystem src/third-party -DRELEASE -DMACOSX -DOSX_SDL2_LIBS -DTILES -DBACKTRACE -DLOCALIZE -DPREFIX=&quot;/nix/store/f070rqbqlqrfklklqc0m93bbxsmqha2j-cataclysm-dda-0.H-2025-07-10-0402&quot; -DDATA_DIR_PREFIX -DUSE_HOME_DIR -Os -Wodr -Werror -Wall -Wextra -Wformat-signedness -Wlogical-op -Wmissing-declarations -Wmissing-noreturn -Wnon-virtual-dtor -Wold-style-cast -Woverloaded-virtual -Wpedantic -Wsuggest-override -Wunused-macros -Wzero-as-null-pointer-constant -Wno-unknown-warning-option -Wno-dangling-reference -Wno-c++20-compat -Wredundant-decls  -g1  -fsigned-char -stdlib=libc++  -std=c++17 -mmacosx-version-min=10.13 -D_THREAD_SAFE -I/nix/store/a1gcik4f5777g7yw2k2znjjmsadlgmcb-sdl2-compat-2.32.64-dev/include -I/nix/store/a1gcik4f5777g7yw2k2znjjmsadlgmcb-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/c16accmdcp1ackda871l4x0b234nhyxv-SDL2_mixer-2.8.1-dev/include/SDL2 -DSDL_SOUND -I/SDL2 -I/nix/store/a1gcik4f5777g7yw2k2znjjmsadlgmcb-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/2yc9y51272dzfd2vyfimwpg328db1p8n-SDL2_image-2.8.8/include/SDL2 -I/nix/store/c16accmdcp1ackda871l4x0b234nhyxv-SDL2_mixer-2.8.1-dev/include/SDL2 -I/nix/store/wki4knqr54n3zqgvl085i5ajhija37gg-SDL2_ttf-2.24.0/include/SDL2 -I/nix/store/a1gcik4f5777g7yw2k2znjjmsadlgmcb-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/2yc9y51272dzfd2vyfimwpg328db1p8n-SDL2_image-2.8.8/include/SDL2 -I/nix/store/c16accmdcp1ackda871l4x0b234nhyxv-SDL2_mixer-2.8.1-dev/include/SDL2 -I/nix/store/wki4knqr54n3zqgvl085i5ajhija37gg-SDL2_ttf-2.24.0/include/SDL2 -D_THREAD_SAFE -I/ -MMD -MP -Winvalid-pch -include-pch pch/main-pch.hpp.pch -c src/mapgendata.cpp -o obj/tiles/mapgendata.o
clang++ -Isrc -isystem src/third-party -DRELEASE -DMACOSX -DOSX_SDL2_LIBS -DTILES -DBACKTRACE -DLOCALIZE -DPREFIX=&quot;/nix/store/f070rqbqlqrfklklqc0m93bbxsmqha2j-cataclysm-dda-0.H-2025-07-10-0402&quot; -DDATA_DIR_PREFIX -DUSE_HOME_DIR -Os -Wodr -Werror -Wall -Wextra -Wformat-signedness -Wlogical-op -Wmissing-declarations -Wmissing-noreturn -Wnon-virtual-dtor -Wold-style-cast -Woverloaded-virtual -Wpedantic -Wsuggest-override -Wunused-macros -Wzero-as-null-pointer-constant -Wno-unknown-warning-option -Wno-dangling-reference -Wno-c++20-compat -Wredundant-decls  -g1  -fsigned-char -stdlib=libc++  -std=c++17 -mmacosx-version-min=10.13 -D_THREAD_SAFE -I/nix/store/a1gcik4f5777g7yw2k2znjjmsadlgmcb-sdl2-compat-2.32.64-dev/include -I/nix/store/a1gcik4f5777g7yw2k2znjjmsadlgmcb-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/c16accmdcp1ackda871l4x0b234nhyxv-SDL2_mixer-2.8.1-dev/include/SDL2 -DSDL_SOUND -I/SDL2 -I/nix/store/a1gcik4f5777g7yw2k2znjjmsadlgmcb-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/2yc9y51272dzfd2vyfimwpg328db1p8n-SDL2_image-2.8.8/include/SDL2 -I/nix/store/c16accmdcp1ackda871l4x0b234nhyxv-SDL2_mixer-2.8.1-dev/include/SDL2 -I/nix/store/wki4knqr54n3zqgvl085i5ajhija37gg-SDL2_ttf-2.24.0/include/SDL2 -I/nix/store/a1gcik4f5777g7yw2k2znjjmsadlgmcb-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/2yc9y51272dzfd2vyfimwpg328db1p8n-SDL2_image-2.8.8/include/SDL2 -I/nix/store/c16accmdcp1ackda871l4x0b234nhyxv-SDL2_mixer-2.8.1-dev/include/SDL2 -I/nix/store/wki4knqr54n3zqgvl085i5ajhija37gg-SDL2_ttf-2.24.0/include/SDL2 -D_THREAD_SAFE -I/ -MMD -MP -Winvalid-pch -include-pch pch/main-pch.hpp.pch -c src/mapgenformat.cpp -o obj/tiles/mapgenformat.o
clang++ -Isrc -isystem src/third-party -DRELEASE -DMACOSX -DOSX_SDL2_LIBS -DTILES -DBACKTRACE -DLOCALIZE -DPREFIX=&quot;/nix/store/f070rqbqlqrfklklqc0m93bbxsmqha2j-cataclysm-dda-0.H-2025-07-10-0402&quot; -DDATA_DIR_PREFIX -DUSE_HOME_DIR -Os -Wodr -Werror -Wall -Wextra -Wformat-signedness -Wlogical-op -Wmissing-declarations -Wmissing-noreturn -Wnon-virtual-dtor -Wold-style-cast -Woverloaded-virtual -Wpedantic -Wsuggest-override -Wunused-macros -Wzero-as-null-pointer-constant -Wno-unknown-warning-option -Wno-dangling-reference -Wno-c++20-compat -Wredundant-decls  -g1  -fsigned-char -stdlib=libc++  -std=c++17 -mmacosx-version-min=10.13 -D_THREAD_SAFE -I/nix/store/a1gcik4f5777g7yw2k2znjjmsadlgmcb-sdl2-compat-2.32.64-dev/include -I/nix/store/a1gcik4f5777g7yw2k2znjjmsadlgmcb-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/c16accmdcp1ackda871l4x0b234nhyxv-SDL2_mixer-2.8.1-dev/include/SDL2 -DSDL_SOUND -I/SDL2 -I/nix/store/a1gcik4f5777g7yw2k2znjjmsadlgmcb-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/2yc9y51272dzfd2vyfimwpg328db1p8n-SDL2_image-2.8.8/include/SDL2 -I/nix/store/c16accmdcp1ackda871l4x0b234nhyxv-SDL2_mixer-2.8.1-dev/include/SDL2 -I/nix/store/wki4knqr54n3zqgvl085i5ajhija37gg-SDL2_ttf-2.24.0/include/SDL2 -I/nix/store/a1gcik4f5777g7yw2k2znjjmsadlgmcb-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/2yc9y51272dzfd2vyfimwpg328db1p8n-SDL2_image-2.8.8/include/SDL2 -I/nix/store/c16accmdcp1ackda871l4x0b234nhyxv-SDL2_mixer-2.8.1-dev/include/SDL2 -I/nix/store/wki4knqr54n3zqgvl085i5ajhija37gg-SDL2_ttf-2.24.0/include/SDL2 -D_THREAD_SAFE -I/ -MMD -MP -Winvalid-pch -include-pch pch/main-pch.hpp.pch -c src/mapsharing.cpp -o obj/tiles/mapsharing.o
clang++ -Isrc -isystem src/third-party -DRELEASE -DMACOSX -DOSX_SDL2_LIBS -DTILES -DBACKTRACE -DLOCALIZE -DPREFIX=&quot;/nix/store/f070rqbqlqrfklklqc0m93bbxsmqha2j-cataclysm-dda-0.H-2025-07-10-0402&quot; -DDATA_DIR_PREFIX -DUSE_HOME_DIR -Os -Wodr -Werror -Wall -Wextra -Wformat-signedness -Wlogical-op -Wmissing-declarations -Wmissing-noreturn -Wnon-virtual-dtor -Wold-style-cast -Woverloaded-virtual -Wpedantic -Wsuggest-override -Wunused-macros -Wzero-as-null-pointer-constant -Wno-unknown-warning-option -Wno-dangling-reference -Wno-c++20-compat -Wredundant-decls  -g1  -fsigned-char -stdlib=libc++  -std=c++17 -mmacosx-version-min=10.13 -D_THREAD_SAFE -I/nix/store/a1gcik4f5777g7yw2k2znjjmsadlgmcb-sdl2-compat-2.32.64-dev/include -I/nix/store/a1gcik4f5777g7yw2k2znjjmsadlgmcb-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/c16accmdcp1ackda871l4x0b234nhyxv-SDL2_mixer-2.8.1-dev/include/SDL2 -DSDL_SOUND -I/SDL2 -I/nix/store/a1gcik4f5777g7yw2k2znjjmsadlgmcb-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/2yc9y51272dzfd2vyfimwpg328db1p8n-SDL2_image-2.8.8/include/SDL2 -I/nix/store/c16accmdcp1ackda871l4x0b234nhyxv-SDL2_mixer-2.8.1-dev/include/SDL2 -I/nix/store/wki4knqr54n3zqgvl085i5ajhija37gg-SDL2_ttf-2.24.0/include/SDL2 -I/nix/store/a1gcik4f5777g7yw2k2znjjmsadlgmcb-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/2yc9y51272dzfd2vyfimwpg328db1p8n-SDL2_image-2.8.8/include/SDL2 -I/nix/store/c16accmdcp1ackda871l4x0b234nhyxv-SDL2_mixer-2.8.1-dev/include/SDL2 -I/nix/store/wki4knqr54n3zqgvl085i5ajhija37gg-SDL2_ttf-2.24.0/include/SDL2 -D_THREAD_SAFE -I/ -MMD -MP -Winvalid-pch -include-pch pch/main-pch.hpp.pch -c src/martialarts.cpp -o obj/tiles/martialarts.o
clang++ -Isrc -isystem src/third-party -DRELEASE -DMACOSX -DOSX_SDL2_LIBS -DTILES -DBACKTRACE -DLOCALIZE -DPREFIX=&quot;/nix/store/f070rqbqlqrfklklqc0m93bbxsmqha2j-cataclysm-dda-0.H-2025-07-10-0402&quot; -DDATA_DIR_PREFIX -DUSE_HOME_DIR -Os -Wodr -Werror -Wall -Wextra -Wformat-signedness -Wlogical-op -Wmissing-declarations -Wmissing-noreturn -Wnon-virtual-dtor -Wold-style-cast -Woverloaded-virtual -Wpedantic -Wsuggest-override -Wunused-macros -Wzero-as-null-pointer-constant -Wno-unknown-warning-option -Wno-dangling-reference -Wno-c++20-compat -Wredundant-decls  -g1  -fsigned-char -stdlib=libc++  -std=c++17 -mmacosx-version-min=10.13 -D_THREAD_SAFE -I/nix/store/a1gcik4f5777g7yw2k2znjjmsadlgmcb-sdl2-compat-2.32.64-dev/include -I/nix/store/a1gcik4f5777g7yw2k2znjjmsadlgmcb-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/c16accmdcp1ackda871l4x0b234nhyxv-SDL2_mixer-2.8.1-dev/include/SDL2 -DSDL_SOUND -I/SDL2 -I/nix/store/a1gcik4f5777g7yw2k2znjjmsadlgmcb-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/2yc9y51272dzfd2vyfimwpg328db1p8n-SDL2_image-2.8.8/include/SDL2 -I/nix/store/c16accmdcp1ackda871l4x0b234nhyxv-SDL2_mixer-2.8.1-dev/include/SDL2 -I/nix/store/wki4knqr54n3zqgvl085i5ajhija37gg-SDL2_ttf-2.24.0/include/SDL2 -I/nix/store/a1gcik4f5777g7yw2k2znjjmsadlgmcb-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/2yc9y51272dzfd2vyfimwpg328db1p8n-SDL2_image-2.8.8/include/SDL2 -I/nix/store/c16accmdcp1ackda871l4x0b234nhyxv-SDL2_mixer-2.8.1-dev/include/SDL2 -I/nix/store/wki4knqr54n3zqgvl085i5ajhija37gg-SDL2_ttf-2.24.0/include/SDL2 -D_THREAD_SAFE -I/ -MMD -MP -Winvalid-pch -include-pch pch/main-pch.hpp.pch -c src/material.cpp -o obj/tiles/material.o
clang++ -Isrc -isystem src/third-party -DRELEASE -DMACOSX -DOSX_SDL2_LIBS -DTILES -DBACKTRACE -DLOCALIZE -DPREFIX=&quot;/nix/store/f070rqbqlqrfklklqc0m93bbxsmqha2j-cataclysm-dda-0.H-2025-07-10-0402&quot; -DDATA_DIR_PREFIX -DUSE_HOME_DIR -Os -Wodr -Werror -Wall -Wextra -Wformat-signedness -Wlogical-op -Wmissing-declarations -Wmissing-noreturn -Wnon-virtual-dtor -Wold-style-cast -Woverloaded-virtual -Wpedantic -Wsuggest-override -Wunused-macros -Wzero-as-null-pointer-constant -Wno-unknown-warning-option -Wno-dangling-reference -Wno-c++20-compat -Wredundant-decls  -g1  -fsigned-char -stdlib=libc++  -std=c++17 -mmacosx-version-min=10.13 -D_THREAD_SAFE -I/nix/store/a1gcik4f5777g7yw2k2znjjmsadlgmcb-sdl2-compat-2.32.64-dev/include -I/nix/store/a1gcik4f5777g7yw2k2znjjmsadlgmcb-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/c16accmdcp1ackda871l4x0b234nhyxv-SDL2_mixer-2.8.1-dev/include/SDL2 -DSDL_SOUND -I/SDL2 -I/nix/store/a1gcik4f5777g7yw2k2znjjmsadlgmcb-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/2yc9y51272dzfd2vyfimwpg328db1p8n-SDL2_image-2.8.8/include/SDL2 -I/nix/store/c16accmdcp1ackda871l4x0b234nhyxv-SDL2_mixer-2.8.1-dev/include/SDL2 -I/nix/store/wki4knqr54n3zqgvl085i5ajhija37gg-SDL2_ttf-2.24.0/include/SDL2 -I/nix/store/a1gcik4f5777g7yw2k2znjjmsadlgmcb-sdl2-compat-2.32.64-dev/include/SDL2 -I/nix/store/2yc9y51272dzfd2vyfimwpg328db1p8n-SDL2_image-2.8.8/include/SDL2 -I/nix/store/c16accmdcp1ackda871l4x0b234nhyxv-SDL2_mixer-2.8.1-dev/include/SDL2 -I/nix/store/wki4knqr54n3zqgvl085i5ajhija37gg-SDL2_ttf-2.24.0/include/SDL2 -D_THREAD_SAFE -I/ -MMD -MP -Winvalid-pch -include-pch pch/main-pch.hpp.pch -c src/math_parser.cpp -o obj/tiles/math_parser.o
src/math_parser.cpp:632:13: error: function &#x27;operator()&#x27; could be declared with attribute &#x27;noreturn&#x27; [-Werror,-Wmissing-noreturn]
  632 |             []( auto /* v */ )
      |             ^
src/math_parser.cpp:765:9: error: function &#x27;operator()&#x27; could be declared with attribute &#x27;noreturn&#x27; [-Werror,-Wmissing-noreturn]
  765 |         []( auto /* v */ )
      |         ^
2 errors generated.
make: *** [Makefile:1057: obj/tiles/math_parser.o] Error 1
make: *** Waiting for unfinished jobs....</pre>
</details>
</details>

--- a/pkgs/games/cataclysm-dda/common.nix
+++ b/pkgs/games/cataclysm-dda/common.nix
@@ -66,6 +66,9 @@ stdenv.mkDerivation {
     patchShebangs lang/compile_mo.sh
   '';
 
+  # remove once on O.I/ahead of upstream commit 15b3cb0
+  env.NIX_CFLAGS_COMPILE = optionalString stdenv.hostPlatform.isDarwin "-Wno-missing-noreturn";
+
   makeFlags = [
     "PREFIX=$(out)"
     "LANGUAGES=all"

--- a/pkgs/games/cataclysm-dda/stable.nix
+++ b/pkgs/games/cataclysm-dda/stable.nix
@@ -17,22 +17,17 @@ let
   };
 
   self = common.overrideAttrs (common: rec {
-    version = "0.H";
+    version = "0.H-2025-07-10-0402";
 
     src = fetchFromGitHub {
       owner = "CleverRaven";
       repo = "Cataclysm-DDA";
-      tag = "${version}-RELEASE";
-      sha256 = "sha256-ZCD5qgqYSX7sS+Tc1oNYq9soYwNUUuWamY2uXfLjGoY=";
+      # Head of 0.H-branch
+      tag = "cdda-${version}";
+      sha256 = "sha256-r4cl8cij68WmQRfg+DHQIeDBIwhgwSre6kAUYZaCPR8=n";
     };
 
     patches = [
-      # fix compilation of the vendored flatbuffers under gcc14
-      (fetchpatch {
-        name = "fix-flatbuffers-with-gcc14";
-        url = "https://github.com/CleverRaven/Cataclysm-DDA/commit/1400b1018ff37196bd24ba4365bd50beb571ac14.patch";
-        hash = "sha256-H0jct6lSQxu48eOZ4f8HICxo89qX49Ksw+Xwwtp7iFM=";
-      })
       # Unconditionally look for translation files in $out/share/locale
       ./locale-path.patch
     ];

